### PR TITLE
Demog schemes placeholders

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/age.json [hunk 1]
```diff
@@ -0,0 +1,1132 @@
+{
+  "SchemeID": "Scheme-f12a1c39",
+  "Name": "Age",
+  "Version": "0.0.0.3",
```
Comment (as2388) [inline code_schemes/age.json]: (0.0.0.4 would sync with the upcoming standards https://github.com/AfricasVoices/Standards/pull/10/files)

File: code_schemes/dadaab_hh_language.json [hunk 2]
```diff
@@ -0,0 +1,142 @@
+{
+  "SchemeID": "Scheme-f503029a",
+  "Name": "Dadaab hh language",
```
Comment (as2388) [inline code_schemes/dadaab_hh_language.json]: Can we call this 'household' from now on. We abbreviated to 'hh' in the past, but that was confusing and unnecessary.

File: code_schemes/dadaab_location.json [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+      "MetaCode": "opt-in",
+      "DisplayText": "opt-in",
+      "NumericValue": -100040,
+      "StringValue": "opt_in",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-SC-a3a065bc",
+      "CodeType": "Meta",
+      "MetaCode": "similar_content",
```
Comment (as2388) [inline code_schemes/dadaab_location.json:126]: thanks for remembering these :)

**Review by as2388 — CHANGES_REQUESTED** [review]

File: code_schemes/age.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/age.json]: updated thanks

File: code_schemes/dadaab_hh_language.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/dadaab_hh_language.json]: Updated on both scheme and file. Will update pipeline configurations and code_schemes in a separate pr.

File: code_schemes/dadaab_location.json (hunk 3 above)
Comment (IsaackMwenda) [reply code_schemes/dadaab_location.json:126]: : )

**Review by as2388 — APPROVED** [review]

**Comment by as2388** [comment]
Requesting review from @lukechurch because some of these schemes are new, and to confirm that we should maintain a household language scheme for each location.

**Comment by lukechurch** [comment]
@IsaackMwenda - I'm a bit unsure about the nationality code - could you send me a pointer to the research design for it?

**Review by lukechurch — APPROVED** [review]

**Pull request merged**